versions.txt

```diff
@@ -3,5 +3,6 @@
 
 mcagar@8.1.0
```

Comment:
What is this file for?

Reply:
It's how we track what version to release come 6.2 as we can't manually bump the package.json because of how lerna works.